src/platform/unix/mod.rs

```diff
@@ -72,6 +68,8 @@ lazy_static! {
     };
 }
 
+static GLOBAL_SHM_COUNT: AtomicUsize = ATOMIC_USIZE_INIT;
```

Comment:
I totally forgot to comment on this: I don't think the `GLOBAL_` prefix is useful. I haven't ever seen something like that used with Rust. (The all-caps already identify it clearly as a global.)

Maybe also good to add a comment that this is used for creating the unique IDs?